de.hamstersimulator.objectsfirst.inspector/src/main/java/de/hamstersimulator/objectsfirst/inspector/model/ClassFactory.java

```diff
@@ -133,7 +133,14 @@ private MethodViewModel createStaticMethodViewModel(final Method method) {
         final Function<List<?>, Object> invokeMethod = params -> {
             try {
                 return method.invoke(null, params.toArray());
-            } catch (final IllegalAccessException | InvocationTargetException e) {
+            } catch (final InvocationTargetException targetEx) {
+                final Throwable cause = targetEx.getCause();
+                if (cause instanceof RuntimeException) {
```

Comment:
Instead of instanceOf checks I prefer dynamic visitors for maintainability. I assume there is one in our utils package. Consider using it.

Reply:
The LambdaVisitor does not support base class checks or default cases, therefore it is not applicable here.  
It is also problematic that the else case would be Exception, but every RuntimeException is also an Exception, therefore we would rely on the registration order, which could make the code more confusing

Reply:
Ok, great. I resolve it. Can you create a feature improvement issue for this?